score_analysis/showbias.py

```diff
@@ -139,6 +139,13 @@ def showbias(
             f"Got unexpected type {type(group_columns)} value for `group_columns`"
         )
 
+    if "threshold" in metric_kwargs:
+        threshold = metric_kwargs["threshold"]
+        if isinstance(threshold, (int, float)):  # Scalar case
+            metric_kwargs["threshold"] = [threshold]
+        elif isinstance(threshold, np.ndarray):  # Numpy array case
+            metric_kwargs["threshold"] = threshold.tolist()
```

Comment:
Two things. You could simply write
```
threshold = np.asarray(metric_kwargs["threshold"])
```
Second, why do you want `threshold` to be a list? It feels more natural to work internally with arrays.

Reply:
Good point. I was just sticking with the convention of using list for `threshold` in `showbias`. But seeing that the rest of `score-analysis` works internally with arrays, I now changed to use arrays in `showbias` as well.